src/arch/armv8/armv8-r/mpu.c

```diff
@@ -546,7 +546,7 @@ void mpu_init()
     bitmap_clear_consecutive(cpu()->arch.profile.mpu.bitmap, 0, mpu_num_entries());
     list_init(&cpu()->arch.profile.mpu.order.list);
 
-    for (mpid_t mpid = 0; mpid < mpu_num_entries(); mpid++) {
+    for (mpid_t mpid = 0; mpid < (ssize_t)mpu_num_entries(); mpid++) {
```

Comment:
The same here.

Reply:
Fixed in aef4397dde5adfee49fdfa9159b05c1b1ccfd493